### PR TITLE
Compile fixes

### PR DESCRIPTION
**Pull request by justinfagnani**
- [ ] CHANGELOG.md has been updated

### CODE REVIEW

**Comment by justinfagnani** [comment]
No tests for these, but we need this fix quick for elements.

**Comment by justinfagnani** [comment]
Merging TBR

File: src/compile-middleware.ts [hunk 1]
```diff
@@ -21,7 +21,27 @@ import {UAParser} from 'ua-parser-js';
...
+  'babel-plugin-transform-es2015-literals',
+  'babel-plugin-transform-es2015-object-super',
+  'babel-plugin-transform-es2015-parameters',
+  'babel-plugin-transform-es2015-shorthand-properties',
+  'babel-plugin-transform-es2015-spread',
+  'babel-plugin-transform-es2015-sticky-regex',
+  'babel-plugin-transform-es2015-template-literals',
+  'babel-plugin-transform-es2015-typeof-symbol',
+  'babel-plugin-transform-es2015-unicode-regex',
+  'babel-plugin-transform-regenerator',
```
Comment (rictic) [inline src/compile-middleware.ts]: Why this list rather than babelLatest?

Comment (justinfagnani) [inline src/compile-middleware.ts]: babel-latest includes a module transform, which assumes that input is an ES6 module and compiles top-level `this` to `undefined` :(

Comment (rictic) [inline src/compile-middleware.ts]: yikes!

**Pull request merged**